src/functions/sendCandidateResults/framework/request-scheduler.ts

```diff
@@ -33,13 +33,20 @@ export class RequestScheduler implements IRequestScheduler {
     @inject(TYPES.IStatusUpdater) private statusUpdater: IStatusUpdater,
   ) {
     this.limiter = new bottleneck({
-      maxConcurrent: null,                 // No limit on concurrent requests
-      minTime: 0,                          // No time waited between each request
-      highWater: 250,                      // Maximum of 250 requests in the queue (max batch size)
-      strategy: bottleneck.strategy.BLOCK, // Ignore any additions to the queue when we reach the max batch size
-      reservoir: 25,                       // Amount of jobs the queue can perform at the start of the queue
-      reservoirRefreshInterval: 1000,      // How often to add new jobs to the queue (every second)
-      reservoirRefreshAmount: 25,          // How many jobs to add to the queue each refresh
+      // No limit on concurrent requests
+      maxConcurrent: null,
+      // No time waited between each request
+      minTime: 0,
+      // Maximum of 250 requests in the queue (max batch size)
+      highWater: this.getNumberFromEnv('NOTIFY_BATCH_SIZE') || 250,
```

Comment:
Should we not be using the config adapter to set these

Reply:
Good point !!!